plugins/processors/dcos_metadata/dcos_metadata.go

```diff
@@ -240,22 +247,26 @@ func (dm *DCOSMetadata) getClient() (*httpcli.Client, error) {
 	return client, nil
 }
 
-// getContainerID retrieves the container ID linked to this task. Task can have
-// multiple statuses. Each status can have multiple container IDs. In DC/OS,
-// there is a one-to-one mapping between tasks and containers; however it is
+// getContainerIDs retrieves the parent container ID (if existent) and the
+// container ID linked to this task. Task can have multiple statuses.
+// Each status can have multiple container IDs. In DC/OS, there is a
+// one-to-one mapping between tasks and containers; however it is
 // possible to have nested containers. Therefore we use the first status, and
-// return its parent container ID if possible, and if not, then its ID.
-func getContainerID(statuses []mesos.TaskStatus) string {
+// return its parent container ID if possible, along with its ID.
```

Comment:
This comment's been rewritten a few times and I'm not sure it's more help than harm now. Can we clear it up, eg:

```suggestion
// getContainerIDs retrieves the container ID and the parent container ID of a                                          
// task from its TaskStatus. The container ID corresponds to the task's                                                 
// container, the parent container ID corresponds to the task's executor's                                              
// container. 
```

Reply:
excellent use of the new github feature! now let's see if this works..

Reply:
![image](https://user-images.githubusercontent.com/5897740/47467166-b3282e80-d7a9-11e8-8941-4b2f379ccd8d.png)